bedrock/wordpress/api.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import print_function, unicode_literals
+from django.conf import settings
+
+import requests
+
+
+def _request(api_url, limit=None, page=1):
+    # 100 is max per page from WP
+    per_page = limit or 100
+    resp = requests.get(api_url, params={'per_page': per_page, 'page': page}, timeout=5)
```

Comment:
I think `resp.raise_for_status()` would be helpful here and combined with the comment below in the exception handling block